atqa-framework/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ts-node": {
+    "files": true
+  },
+  "compilerOptions": {
+    "target": "es2018",
+    "lib": ["es2016", "es2017", "es2018", "es2019", "es2020", "dom"],
```

Comment:
AFAIK you don't need to specify every version, only the latest one

Reply:
`es2020` includes `es2019`, `es2019` includes `es2018`, they're based on top of each other. You can just specify `es2022` here and in target